src/diff/props.js

```diff
@@ -133,10 +133,7 @@ export function setProperty(dom, name, value, oldValue, isSvg) {
 
 		if (typeof value === 'function') {
 			// never serialize functions as attribute values
-		} else if (
-			value != null &&
-			(value !== false || (name[0] === 'a' && name[1] === 'r'))
-		) {
+		} else if (value != null && (value !== false || name.indexOf('-') != -1)) {
```

Comment:
I wonder if we should invert these two conditions now.

Reply:
You mean switch it with the remove-path? I think we would add more code in differentiating 0 and empty string

Reply:
I'm good with the current way.